packages/tables/resources/lang/de/table.php

```diff
@@ -102,7 +102,7 @@
 
     'empty' => [
 
-        'heading' => 'Keine Datensätze gefunden',
+        'heading' => 'Keine :model gefunden',
```

Comment:
```suggestion
        'heading' => 'Keine :model',
```

Reply:
That's even shorter and nearer on the english term